src/encord_active/app/data_quality/sub_pages/explorer.py

```diff
@@ -138,9 +138,9 @@ def fill_data_quality_window(
     if (embedding_information.type == EmbeddingType.CLASSIFICATION) and len(embedding_information.collections) == 0:
         st.write("Image-level embedding file is not available for this project.")
         return
-    if (embedding_information.type == EmbeddingType.OBJECT) and len(embedding_information.collections) == 0:
-        st.write("Object-level embedding file is not available for this project.")
-        return
+    # if (embedding_information.type == EmbeddingType.OBJECT) and len(embedding_information.collections) == 0:
+    #     st.write("Object-level embedding file is not available for this project.")
+    #     return
```

Comment:
Remove if not useful?

Reply:
it is actually an important part, I commented it out during some of my tests.